### PR TITLE
Destroy Method

### PR DESCRIPTION
**Pull request by arrudagates**

### CODE REVIEW

File: core/src/generator/bindings.rs [hunk 1]
```diff
@@ -140,6 +157,15 @@ impl BindingGenerator {
         content
     }
 
+    /// Generate source file epilogue.
```
Comment (notdanilo) [inline core/src/generator/bindings.rs:160]: Fix doc

File: core/src/generator/bindings.rs [hunk 2]
```diff
@@ -148,7 +174,12 @@ impl BindingGenerator {
         for item in &implementation.items {
             match item {
                 Constant(_) => Logger::log("Const extern not supported."),
-                Method(method) => content.push_line(self.generate_function(context, implementation, method))
+                Method(method) => {
+                    content.push_line(self.generate_function(context, implementation, method));
+                    content.push_line(BindingGenerator::generate_destroy(
+                        &implementation.self_.name,
+                    ))
```
Comment (notdanilo) [inline core/src/generator/bindings.rs:181]: This is repeated once per method?

File: core/src/generator/externs.rs [hunk 3]
```diff
@@ -90,24 +98,43 @@ impl ExternGenerator {
...
-        })
+        let mut tokens =
+            implementation
+                .items
+                .iter()
+                .fold(TokenStream::new(), |mut tokens, item| match item {
+                    Constant(_) => unimplemented!("Constants aren't implemented yet."),
+                    Method(method) => {
+                        let method = Self::generate_function(context, implementation, method);
+                        println!("{}", method);
```
Comment (notdanilo) [inline core/src/generator/externs.rs:132]: Would you please removed this print fn I left here?

**Review by notdanilo — COMMENTED** [review]
Just some minor requests.

**Comment by arrudagates** [comment]
Requests done.

**Review by notdanilo — APPROVED** [review]

**Pull request merged**